cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,39 @@
+/* eslint-disable */
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/generateUser')
 describe('Student Registration page', () => {
   before(() => {
+    cy.visit('https://demoqa.com/automation-practice-form')
+  })
 
-  });
+  it('should send the form with filled fields and check modal', () => {
+    const { userName, email, mobile, subject, currentAddress, state, city } =
+      generateUser()
 
-  it('', () => {
+    cy.findById('firstName').type(userName)
+    cy.findById('lastName').type(userName)
+    cy.findById('userEmail').type(email)
+    cy.findById('userNumber').type(mobile)
+    cy.findById('currentAddress').type(currentAddress)
+    cy.findById('state').type(state)
+    cy.findById('city').type(city)
```

Comment:
Replace `cy.findById` with `cy.get` for these lines as well. For example, use `cy.get('#userEmail')` for the email field.